Podcast/BookmarkViewController.swift

```diff
@@ -135,7 +135,10 @@ class BookmarkViewController: DiscoverComponentViewController, EmptyStateTableVi
     func bookmarkTableViewCellDidPressRecommendButton(bookmarksTableViewCell: BookmarkTableViewCell) {
         guard let episodeIndexPath = bookmarkTableView.indexPath(for: bookmarksTableViewCell) else { return }
         let episode = episodes[episodeIndexPath.row]
-        episode.recommendedChange(completion:  bookmarksTableViewCell.setRecommendedButtonToState)
+        editRecastAction(episode: episode, completion:
+            { (_,_) in
```

Comment:
i think you can just do `_ in` here

Reply:
actually I can't do that but i can remove the parenthesis.